contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -520,34 +525,17 @@ func (s *SchedulerServer) prepareStaticPods() (data []byte, staticPodCPUs, stati
 	return
 }
 
-// TODO(jdef): hacked from kubelet/server/server.go
-// TODO(k8s): replace this with clientcmd
-func (s *SchedulerServer) createAPIServerClient() (*clientset.Clientset, error) {
-	authInfo, err := clientauth.LoadFromFile(s.authPath)
-	if err != nil {
-		log.Warningf("Could not load kubernetes auth path: %v. Continuing with defaults.", err)
-	}
-	if authInfo == nil {
-		// authInfo didn't load correctly - continue with defaults.
-		authInfo = &clientauth.Info{}
-	}
-	clientConfig, err := authInfo.MergeWithConfig(client.Config{})
+// TODO(jdef): hacked from plugin/cmd/kube-scheduler/app/server.go
+func (s *SchedulerServer) createAPIServerClientConfig() (*client.Config, error) {
+	kubeconfig, err := clientcmd.BuildConfigFromFlags(s.apiServerList[0], s.kubeconfig)
 	if err != nil {
 		return nil, err
 	}
-	if len(s.apiServerList) < 1 {
-		return nil, fmt.Errorf("no api servers specified")
-	}
-	// TODO: adapt Kube client to support LB over several servers
-	if len(s.apiServerList) > 1 {
-		log.Infof("Multiple api servers specified.  Picking first one")
-	}
-	clientConfig.Host = s.apiServerList[0]
-	c, err := clientset.NewForConfig(&clientConfig)
-	if err != nil {
-		return nil, err
-	}
-	return c, nil
+
+	// Override kubeconfig qps/burst settings from flags
+	kubeconfig.QPS = s.kubeAPIQPS
```

Comment:
Thinking that this is the magic that really made conformance pass -- maybe the "send events" machinery has a dep on this config being non-zero now?. Perhaps convert the scheduler back to using clientset instead of `client/unversioned` and re-rest? After all, the kubelet sends events using clientset.